tests/scalapack/scalapack_06.cc

```diff
@@ -44,26 +45,24 @@ void test(const unsigned int size, const unsigned int block_size, const NumberTy
 
   ConditionalOStream pcout (std::cout, (this_mpi_process ==0));
 
+  std::shared_ptr<Utilities::MPI::ProcessGrid> grid = std::make_shared<Utilities::MPI::ProcessGrid>(mpi_communicator,size,size,block_size,block_size);
+
+  pcout << size << " " << block_size << std::endl;
```

Comment:
maybe keep output of process grid rows/cols?

Reply:
Done!